x/market/handler/handler_test.go

```
@@ -488,3 +1427,26 @@ func (st *testSuite) createProvider(attr attr.Attributes) ptypes.Provider {
 
 	return prov
 }
+
+func (st *testSuite) createDeployment() (dv1.Deployment, dtypes.Groups) {
```

golangci-lint (2.5.0)

[error] func (*testSuite).createDeployment is unused

(unused)